scene/gui/color_picker.cpp

```diff
@@ -383,8 +387,17 @@ void ColorPicker::_value_changed(double) {
 		last_color = color;
 	}
 
-	_set_pick_color(color, false);
-	emit_signal(SNAME("color_changed"), color);
+	set_pick_color(color);
```

Comment:
Why `_set_pick_color(color, false);` was changed? This is a callback for slider value changed, so updating sliders again is redundant.